src/charging-station/ChargingStationWorker.ts

```diff
@@ -32,6 +33,7 @@ if (Utils.workerPoolInUse()) {
  */
 function addMessageListener(): void {
   parentPort?.on('message', (message: ChargingStationWorkerMessage) => {
+    logger.debug(`${logPrefix()} ${JSON.stringify(message)}`);
```

Comment:
Add a bit of information about the payload shown: main >> worker id or worker id >> main? ...

And log the internal messaging payload in both direction.

Reply:
By `worker id` I assume you mean `threadId` ?
And for `main` you mean `message` ?